src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(health, name) {
+    this.health = 100;
```

Comment:
The constructor of the Animal class should not accept the 'health' parameter since health is always initialized to 100. You should remove the 'health' parameter from the constructor signature.